code/modules/vehicle/ambulance.dm

```diff
@@ -0,0 +1,55 @@
+/obj/vehicle/ambulance
+	name = "ambulance"
+	desc = "what the paramedic uses to run over people to take to medbay."
+	icon_state = "docwagon2"
+	keytype = /obj/item/key/ambulance
+
+
+/obj/item/key/ambulance
+	name = "ambulance key"
+	desc = "A keyring with a small steel key, and tag with a red cross on it."
+	icon_state = "keydoc"
+
+
+/obj/vehicle/ambulance/handle_vehicle_offsets()
+	..()
+	if(buckled_mob)
+		switch(buckled_mob.dir)
+			if(SOUTH)
+				buckled_mob.pixel_x = 0
+				buckled_mob.pixel_y = 7
+			if(WEST)
+				buckled_mob.pixel_x = 13
+				buckled_mob.pixel_y = 7
+			if(NORTH)
+				buckled_mob.pixel_x = 0
+				buckled_mob.pixel_y = 4
+			if(EAST)
+				buckled_mob.pixel_x = -13
+				buckled_mob.pixel_y = 7
+
+
+/obj/vehicle/ambulance/attackby(obj/item/I, mob/user, params)
+	//add ambulance bed hookup here
+	..()
+
+/obj/vehicle/ambulance/RunOver(var/mob/living/carbon/human/H)
```

Comment:
Let this die, IMO. It's annoying that an emergency vehicle deals damage when you accidentally run over a patient on the ground.